README.md

```diff
@@ -2,10 +2,36 @@
 
 # Claim tax benefits / Réclamer des avantages fiscaux
 
-| en                                                                                                                                                                                                         | fr                                                                                                                                                                                                                           |
-| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| This is a small frontend to trial user flows for a future CRA service that will help Canadians receive the benefits to which they are entitled.                                                            | Voici une petite interface qui nous sert à tester des parcours utilisateur pour un futur service de l'ARC qui aidera les Canadiens à recevoir les prestations auxquelles ils ont droit.                                      |
-| It’s a server-side [express](https://expressjs.com/) application using [Pug](https://pugjs.org/api/getting-started.html) templating on the server and schnazzy [SCSS](https://sass-lang.com/) stylesheets. | Il s'agit d'une application [express](https://expressjs.com/) qui utilise les modèles [Pug](https://pugjs.org/api/getting-started.html) sur le serveur ainsi que des chics feuilles de style [SCSS](https://sass-lang.com/). |
+This is a demo of a service to make tax filing faster and easier for a preselected group of eligible low-income Canadians. This service was designed and developed by the [Canadian Digital Service (CDS)](https://digital.canada.ca/) in partnership with the [Canada Revenue Agency (CRA)](https://www.canada.ca/en/revenue-agency.html).
+
+Much of the information needed to file a tax return is already known to CRA. At the same time, low-confidence tax filers are very worried about making mistakes when submitting a tax return. The concept behind this demo application is to provide a wizard-like tax filing experience: presenting the tax filer with information that CRA already has about them and asking them to confirm it rather than having them type it in.
+
+While this repository is no longer actively maintained, the code is open and available for demonstration purposes or reuse. In addition to an end-to-end flow, it is an example of best-practice development for an API-driven cloud-native frontend application.
```

Comment:
perhaps a bit nitpicky, but can I get a comma before and after could-native? You're free to disagree as I realize this is a preference of mine

Reply:
I'll meet you halfway.